Test/ElasticsearchTestCase.php

```diff
@@ -11,239 +11,11 @@
 
 namespace ONGR\ElasticsearchBundle\Test;
 
-use Elasticsearch\Common\Exceptions\ElasticsearchException;
-use ONGR\ElasticsearchBundle\Client\Connection;
-use ONGR\ElasticsearchBundle\ORM\Manager;
-use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
-use Symfony\Component\DependencyInjection\ContainerInterface;
-
 /**
- * Base test which creates unique connection to test with.
+ * Class ElasticsearchTestCase.
```

Comment:
Comment too general. We need to disallow usage of `Class` word in doc summaries...